pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -236,6 +268,8 @@ type Controller struct {
 	// workers that are responsible for tainting nodes.
 	zoneNoExecuteTainter map[string]*scheduler.RateLimitedTimedQueue
 
+	retryMarkPodsNotReadySet *retryMarkPodsNotReadySet
```

Comment:
I'm not a fan of the name, though i don't have a good candidate.
nodesToRetry maybe?